### PR TITLE
added mark as complete action

### PR DESCRIPTION
**Pull request by Sanjit-Prasad**
1. Added mark as complete action, when clicked it strikes down the todo_item.
2. Added due_date for each todo_item.
3. Added priority selection shown using prompt.
Remaining task: to add styling in css and java-scripts.

### CODE REVIEW

**Comment by achheSharma** [comment]
It is a good practice to Add screenshots of all the visual changes you've made in the description

File: app/models/todo_item.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class TodoItem < ApplicationRecord
   belongs_to :todo_list
-end
+def completed?
+   !completed_at.blank?
```
Comment (achheSharma) [inline app/models/todo_item.rb:4]: better use completed_at.present?

File: db/migrate/20200512132026_add_priority_to_todo_items.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddPriorityToTodoItems < ActiveRecord::Migration[6.0]
+  def change
+    add_column :todo_items, :priority, :string
```
Comment (achheSharma) [inline db/migrate/20200512132026_add_priority_to_todo_items.rb:3]: String priority columns, Are you sure this is a good way to go?
Will you be able to order items on the basis of their priority?

File: app/models/todo_item.rb (hunk 1 above)
Comment (Sanjit-Prasad) [reply app/models/todo_item.rb:4]: okay sir, update with that.

File: db/migrate/20200512132026_add_priority_to_todo_items.rb (hunk 2 above)
Comment (Sanjit-Prasad) [reply db/migrate/20200512132026_add_priority_to_todo_items.rb:3]: Changed it to integer and was able to sort it using,
	default_scope { order({completed_at: :asc}, {priority: :asc}, {created_at: :desc}) }
in the "app/model/todo_item.rb".
But the problem is I want to display it as string on page, like 1 => VeryHigh, 2=> High, ... , 5 => VeryLow. But couldn't figure it how?

**Pull request merged**